packages/relay/src/lib/errors/MirrorNodeClientError.ts

```diff
@@ -23,7 +23,7 @@ export class MirrorNodeClientError extends Error {
     public statusCode: number;
     public errorMessage?: string;
 
-    static retryErrorCodes: Array<number> = [400, 404, 408, 425, 500]
+    static retryErrorCodes: Array<number> = [404]; // we should only retry on 404 errors
```

Comment:
This is currently only used in `MirroNodeClient.ts`.
Move it there, make it nonstatic and set it via an env config.
Then it's easy to configure whiles we explore the right UX for users.

Default can be [404] if the env variable is empty. If not parse it and set the array appropriately

Reply:
✅  Done. 
I have added documentation for that property and also changed my recommended values to test initially as:
```
MIRROR_NODE_RETRIES = 0
MIRROR_NODE_RETRY_CODES=[]
```